tests/test_database/test_repositories/test_base_sqlite_repository.py

```
@@ -0,0 +1,785 @@
+        :param repository: Repository instance
+        :returns: None
+        """
+        # Check the session configuration
+        assert repository.SessionLocal is not None
+        session = repository.SessionLocal()
+        try:
+            assert session.expire_on_commit is False
+        finally:
+            session.close()
+
+    def test_model_type_generic(self, repository):
```

Ruff (0.13.1)

Unused method argument: `repository`

(ARG002)